tasks/main.yml

```diff
@@ -1,12 +1,13 @@
+- include: configure-swift-backups.yml
+  when: "{{ OPENCRAFT_BACKUP_SWIFT_ENABLED }}"
+
 - name: Set login shell for user www-data
   user: name=www-data shell=/bin/bash
 
-- name: "Set permissions on {{ www_data_home_dir }}"
+- name: Create {{ www_data_home_dir }}
   file: path="{{ www_data_home_dir }}" owner=www-data group=www-data mode=750 state=directory
 
-- name: Clone the opencraft repository
-  git: repo=https://github.com/open-craft/opencraft.git dest="{{ opencraft_root_dir }}" update=no
-  become_user: www-data
```

Comment:
Don't we still need this `become_user` statement when cloning the opencraft.git repo?